src/prettytable/prettytable.py

```diff
@@ -1399,7 +1419,7 @@ def _set_orgmode_style(self) -> None:
     def _set_markdown_style(self) -> None:
         self.header = True
         self.border = True
-        self._hrules = None
+        self._hrules = HRuleStyle.HEADER
```

Comment:
`None` is not a "valid" value. The closest would be `NONE`, however that removes the header completely. Using `HRuleStyle.HEADER` doesn't require any test changes and is fully compatible with the previous `None`. https://github.com/jazzband/prettytable/pull/327/commits/06a644261ca7e0e7e2559e1cfe034d92dc9c4150

https://github.com/jazzband/prettytable/blob/d80f5234a7dc27c72c1c080b6afb76ab131cc89a/tests/test_prettytable.py#L1622-L1631